### PR TITLE
Fix/multilang feed links

### PR DESCRIPTION
**Pull request by ChristianFrisson**
### Description

Hi ablog team!

This PR corrects atom feed links for multilang blogs.

Example of [en/archives/atom.xml](https://sat-metalab.gitlab.io/-/blog/-/jobs/1830086071/artifacts/public/en/archives/atom.xml) file contents without this PR on commit 61cdb55 (generated with [this gitlab pipeline](https://gitlab.com/sat-metalab/blog/-/commit/117c092a9ce8cc7a8b46191a5fa01d9612788634/pipelines?ref=test%2Fablog-fork-main)):
```
...
<link href="https://sat-metalab.gitlab.io/blog/archives/atom.xml" rel="self"/>
...
<link href="https://sat-metalab.gitlab.io/blog/2021/first-post.html" rel="alternate"/>
...
```

Example of [en/archives/atom.xml](https://sat-metalab.gitlab.io/-/blog/-/jobs/1830024159/artifacts/public/en/archives/atom.xml) file contents with this PR on commit 45f5bd7 (generated with [this gitlab pipeline](https://gitlab.com/sat-metalab/blog/-/commit/6bda6d10c9aec0368a198804b22b28bba5160f80/pipelines?ref=test%2Fablog-fork-fix-multilang-feed-links)):
```
...
<link href="https://sat-metalab.gitlab.io/blog/en/archives/atom.xml" rel="self"/>
...
<link href="https://sat-metalab.gitlab.io/blog/en/2021/first-post.html" rel="alternate"/>
...
```
If a blog is configured with a single language, this PR still generates an `archives/atom.xml` file with links without lang substring, as it was already the case before this PR, and as in the first example above.

### CODE REVIEW

**Comment by ChristianFrisson** [comment]
The unsuccessful checks blocked by (@azure-pipelines
sunpy.ablog — Build #sunpy.ablog_20211129.02 failed) seem to be independent of this PR:
https://dev.azure.com/sunpy/ablog/_build/results?buildId=21653&view=results
```
No hosted parallelism has been purchased or granted. To request a free parallelism grant, please fill out the following form https://aka.ms/azpipelines-parallelism-request
```
From what I understand, this [Azure DevOps Parallelism Request](https://aka.ms/azpipelines-parallelism-request) might need to be filled by the ablog owners.

**Comment by nabobalis** [comment]
Thanks for the PR. I will review it soon.

The request has been filed, but we are waiting for a response.

File: ablog/post.py [hunk 1]
```diff
@@ -666,13 +666,17 @@ def generate_atom_feeds(app):
     if not url:
         return
 
+    base_url = url
+    if len(blog.blog_languages) > 1:
```
Comment (nabobalis) [inline ablog/post.py:670]: Does `blog.blog_languages` return a list of languages?

Comment (ChristianFrisson) [reply ablog/post.py:670]: `blog.blog_languages` returns a dict of languages: https://ablog.readthedocs.io/en/latest/manual/ablog-configuration-options/#confval-blog_languages.

Comment (nabobalis) [reply ablog/post.py:670]: Thanks.

**Comment by nabobalis** [comment]
The build fails are something I need to patch, otherwise this looks good to me.

**Review by nabobalis — APPROVED** [review]

**Comment by ChristianFrisson** [comment]
> The build fails are something I need to patch, otherwise this looks good to me.

Thanks @nabobalis for having merged this proposal!

**Comment by nabobalis** [comment]
No problem. I will try to get this out in a new release soon.

**Pull request merged**